teps/0001-tekton-enhancement-proposal-process.md

```diff
@@ -0,0 +1,473 @@
+    this TEP in their `superseded-by`.
+  * In the form `TEP-123`
+* **superseded-by** Optional
+  * A list of TEPs that supersede this TEP. Use of this should be
+    paired with this TEP moving into the `Replaced` status.
+  * In the form `TEP-123`
+
+
+### TEP Workflow
+
+A TEP has the following states
+
+- `proposed`: The TEP has been proposed and is actively being
+  defined. This is the starting state while the TEP is being fleshed
+  out and actively defined and discussed.
+- `implementable`: The approvers have approved this TEP for
+  implementation.
+- `implemented`: The TEP has been implemented and is no longer
+  actively changed. From that point on, the TEP should be considered
+  *read-only*.
+- `withdrawn`: The TEP has been withdrawn by the authors or by the
+  community on agreement with the authors.
+- `replaced`: The TEP has been replaced by a new TEP. The
+  `superseded-by` metadata value should point to the new TEP.
+
+When a TEP is merged with the `proposed` state, this means the
+project owners acknowledge this is something we need to work on *but*
+the proposal needs to be more detailed before we can go ahead and
+implement it in the main project(s). This state doesn't prevent using
+`tektoncd/experimental` to *experiment* and gather feedback.
+
+A TEP can be created with the `implementable` state if it doesn't need
+any more discussion and got approved as it.
+
+See [Examples](#examples) to see examples of TEP workflow on use cases.
+
+### Git Implementation
+
+TEPs are checked into the community repo under the `/teps` directory.
+
+New TEPs can be checked in with a file name in the form of
+`draft-YYYYMMDD-my-title.md`. As significant work is done on the TEP,
+the authors can assign a TEP number. No other changes should be put in
+that PR so that it can be approved quickly and minimize merge
+conflicts. The TEP number can also be done as part of the initial
+submission if the PR is likely to be uncontested and merged quickly.
+
+### Prior Art
+
+The TEP process as proposed was essentially adapted from the
+[Kubernetes KEP process][], which itself is essentially stolen from the [Rust RFC
+process][] which itself seems to be very similar to the [Python PEP
+process][]
+
+[Rust RFC process]: https://github.com/rust-lang/rfcs
+[Kubernetes KEP process]: https://github.com/kubernetes/enhancements/tree/master/keps
+[Python PEP process]: https://www.python.org/dev/peps/
+
+
+## Examples
+
+Let's give some example of workflow to give reader a better
+understanding on how and when TEP should be created and how they are
+managed across time.
+
+Those are examples, and do not necessarily reflect what happened, or
+will happens on the particular subject they are about. They are here
+to give more context and ideas on different situation that could rise
+while following the TEP process.
+
+### Share Task and Pipeline as OCI artifact
+
+For more context, this is linked to the following:
```

Comment:
im not sure what this sentence is trying to say, maybe "see the following for more context"?